pingdom/src/main/java/com/pulse/dbcp/DataBaseUtility.java

```diff
@@ -0,0 +1,51 @@
+package com.pulse.dbcp;
+
+import org.apache.commons.dbcp2.BasicDataSource;
+
+public class DataBaseUtility {
+
+
+    private static final String DRIVER_CLASS_NAME = "org.postgresql.Driver";
```

Comment:
@nandini2052 experimented something to move the DB URL, name, password to properties. Why don't you check it out for this also?